docs/en/sql-reference/aggregate-functions/reference/rankCorr.md

```diff
@@ -0,0 +1,51 @@
+## rankCorr {#agg_function-rankcorr}
+
+The aggregate function which computes a rank correlation coefficient.
+
+**Syntax**
+
+``` sql
+rankCorr(x, y)
+```
+
+**Parameters**
+
+-   `x` — Floating-point number. [Float64](../../../sql-reference/data-types/float.md#float32-float64).
```

Comment:
Wrong.

Reply:
Also it is possible to write something more informative in the description.

Reply:
> Wrong.

Incorrect type specified?